.test-infra/jenkins/job_PostCommit_Java11_ValidatesRunner_PortabilityApi_Dataflow.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_PortabilityApi_Dataflow',
+  'Run Dataflow PortabilityApi ValidatesRunner with Java 11', 'Google Cloud Dataflow Runner PortabilityApi ValidatesRunner Java 11 Tests', this) {
+
+  description('Runs the ValidatesRunner suite on the Java 11 enabled Dataflow PortabilityApi runner.')
+
+  commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 180)
+
+  publishers {
+    archiveJunit('**/build/test-results/**/*.xml')
+  }
+
+  steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
+      tasks(':beam-runners-google-cloud-dataflow-java:validatesRunnerFnApiWorkerTest')
+      switches ('-Pdockerfile=Dockerfile-java11')
```

Comment:
One more thing, because I'm not sure: we're using docker container with Java11. Shouldn't gradle also run with java 11? Such as we specified in `job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy`?

Reply:
No, that's not the case here. By supplying a Java 11 Docker image, we ensure that during execution on Dataflow, there's only JRE 11 available. The tests are executed on Dataflow runners, so running gradle with Java 11 is not necessary

Reply:
Ok - makes sense. Thanks for providing the details.